MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2548,7 +2528,7 @@ public MediaStream GetMediaStream(IEnumerable<MediaStream> allStream, int? desir
                         scaleVal));
             }
 
-            return filters;
+            return filters.ToString();
```

Comment:
This won't work I think. You should get rid of the list entirely and just return in every `if`